src/Controller/ItemController.php

```diff
@@ -1097,7 +1097,12 @@ public function stepperAction($roomId, $itemId, Request $request, ItemService $i
         $rubricManager->select();
         $itemList = $rubricManager->get();
         $items = $itemList->to_array();
-        
+        foreach ($items as $key => $temp) {
```

Comment:
Instead of post filtering the items, can you please check if setting a limit on `$rubricManager` does also work?

What I mean is using `$rubricManager->setInactiveEntriesLimit(cs_manager::SHOW_ENTRIES_ONLY_ACTIVATED)`